### PR TITLE
Bookings tabular data #26

### LINKED ISSUE #26: User sees tabular data for bookings
* People booked within past week
* People released on bond within past week

### PR DESCRIPTION
**Pull request by hartsick**
Adds 'Current Week' heading under 'Bookings Summary', with two data summaries:

1. number of people booked in last week
1. for people booked in last week, number of people released and its percent of total

Closes #26

### CODE REVIEW

File: spec/models/booking_spec.rb [hunk 1]
```diff
@@ -12,6 +14,22 @@
...
   	end
+
+    it 'should return inactive bookings' do
+      FactoryGirl.create(:booking)
+      FactoryGirl.create(:booking, :inactive)
+
+      expect(Booking.inactive.count).to eq(1)
+    end
+
+    it 'should return all bookings in last week' do
```
Comment (hartsick) [inline spec/models/booking_spec.rb]: Have fixed entire file — think Github is just confused and hasn't closed the comment. 
¯\_(ツ)_/¯

File: spec/features/bookings_spec.rb [hunk 2]
```diff
@@ -0,0 +1,31 @@
+require 'rails_helper'
+
+describe 'bookings data' do
+  before(:each) do
+    login_as FactoryGirl.create(:user)
+  end
+
+  it 'shows count of current and inactive bookings from past week', js: true do
```
Comment (graysonwright) [inline spec/features/bookings_spec.rb]: Don't think this spec requires `js: true` - can we remove that to speed up our test suite?

File: spec/features/bookings_spec.rb [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+    FactoryGirl.create_list(:booking, 2, :last_week, :inactive)
+
+    visit '/'
+
+    booked_current_week = page.find('.booking-stats .current-week-booked').text
+
+    expect(booked_current_week).to eq('4 people booked within past week')
+  end
+
+  it 'shows count of people released on bond in past week', js: true do
```
Comment (graysonwright) [inline spec/features/bookings_spec.rb]: Same here.

File: spec/models/booking_spec.rb [hunk 4]
```diff
@@ -1,17 +1,35 @@
 require 'rails_helper'
 
 RSpec.describe Booking, type: :model do
-  it "has a valid factory" do
+  it 'has valid factory and traits' do
     expect(FactoryGirl.create(:booking)).to be_valid
+    expect(FactoryGirl.create(:booking, :inactive)).to be_valid
+    expect(FactoryGirl.create(:booking, :last_week)).to be_valid
```
Comment (graysonwright) [inline spec/models/booking_spec.rb]: 🆒

File: spec/models/booking_spec.rb [hunk 5]
```diff
@@ -1,17 +1,35 @@
...
+
+    it 'should return inactive bookings' do
+      FactoryGirl.create(:booking)
+      FactoryGirl.create(:booking, :inactive)
+
+      expect(Booking.inactive.count).to eq(1)
+    end
+
+    it 'should return all bookings in last week' do
+      FactoryGirl.create(:booking, booking_date_time: DateTime.now - 8.days)
```
Comment (graysonwright) [inline spec/models/booking_spec.rb]: How do you feel about using `8.days.ago`? It's a bit more concise.

File: spec/features/bookings_spec.rb (hunk 3 above)
Comment (hartsick) [inline spec/features/bookings_spec.rb]: Good catch!

**Pull request merged**